src/install.js

```diff
@@ -56,11 +57,19 @@ export const DEPENDENCY_FIELDS = [
  * @param  {String} name - name of the dependency.
  * @return {Observable} - observable sequence of `package.json` objects.
  */
-export function resolveLocal (nodeModules, parentTarget, name) {
+export function resolveLocal (nodeModules, parentTarget, name, version) {
 	const linkname = path.join(nodeModules, parentTarget, 'node_modules', name)
 	const fetch = () => EmptyObservable.create()
 	log(`resolving ${linkname} from node_modules`)
 
+	// support `file:` with symlinks
+	if (version.substr(0, 5) === 'file:') {
```

Comment:
I agree with @mgcrea here. If possible, we should avoid adding unneeded dependencies here. This is a simple enough check. Seems like an overkill to use `npm-package-arg` instead.

I was originally thinking that `url.parse('file:///').protocol` might be a better fit, but since there is only one check here, I like this solution better. 👍